package.json

```
@@ -131,7 +131,7 @@
 		"@wordpress/data-controls": "2.2.7",
 		"@wordpress/dependency-extraction-webpack-plugin": "3.2.1",
 		"@wordpress/dom": "3.2.7",
-		"@wordpress/e2e-test-utils": "6.0.1",
+		"@wordpress/e2e-test-utils": "^6.0.2",
```

We should lock the dep
```suggestion
		"@wordpress/e2e-test-utils": "6.0.2",
```